### ISSUE TITLE
Can we use z-addr for mining?

### ISSUE THREAD

**Issue by DinoStray** [zcash/zcash#3359]
Can we use z-addr for mining?

**Comment by garethtdavies** [comment]
No, it is part of the protocol and summarised here: https://github.com/zcash/zcash/wiki/Mining-Guide 

> Coins are mined into a t-addr (transparent address), but can only be spent to a z-addr (shielded address), and must be swept out of the t-addr in one transaction with no change. 

If you are mining to a pool, it is possible that the pool mines the block (to a t-addr), shields it and then distributes to z-addrs. However, this isn't done on any pools that I am aware of currently due to performance reasons but this would be likely to change in the short/medium term.

**Comment by str4d** [comment]
If [ZIP 213](https://github.com/zcash/zips/pull/217) is accepted and deployed in NU3 (targeted for activation April 2020), then you would be able to mine directly to a z-address.
